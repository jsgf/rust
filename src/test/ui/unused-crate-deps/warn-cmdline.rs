// Check for unused crate dep, no path

// edition:2018
// check-pass
// compile-flags: -Wunused-crate-deps
// aux-crate:bar=bar.rs
// no-prefer-dynamic
// ignore-tidy-linelength

fn main() {}
//~^ WARNING External crate `bar` unused in `warn_cmdline`. Remove the dependency or add `use bar as _;`.
