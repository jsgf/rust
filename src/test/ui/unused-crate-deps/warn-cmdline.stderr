warning: External crate `bar` unused in `warn_cmdline`. Remove the dependency or add `use bar as _;`.
  --> $DIR/warn-cmdline.rs:10:1
   |
LL | fn main() {}
   | ^
   |
   = note: requested on the command line with `-W unused-crate-deps`

